### PR TITLE
Add BSC docs

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -138,6 +138,17 @@ module.exports = {
                             '/operations/bitcoin/tools',
                         ]
                     },
+                    {
+                        title: 'Binance Smart Chain',
+                        collapsable: true,
+                        children: [
+                            '/operations/binance/',
```
Comment (easeev) [inline docs/.vuepress/config.js:145]: I suggest `/bsc` handle instead of `/binance` to avoid any confusion with Binance Chain

Comment (akegaviar) [reply docs/.vuepress/config.js]: Done

File: docs/operations/bsc/types.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    content: Learn the difference between a shared and a dedicated Binance Smart Chain node deployed with the Chainstack managed blockchain services.
+  - name: keywords
+    content: bsc binance node shared dedicated blockchain
+---
+
+# Types
+
+Chainstack supports deploying a Binance Smart Chain node of the following types:
+
+* Shared — the node resources are shared with other platform users.
```
Comment (neewy) [inline docs/operations/bsc/types.md:13]: We should mention included JSON-RPC request volumes here

**Review by neewy — CHANGES_REQUESTED** [review]
Amazing tutorial! Having in mind our shared nodes service changes, should we mention it for BSC too?

**Comment by CaJIbHuK** [comment]
@akegaviar @easeev what shall we do with this PR?

**Review by easeev — DISMISSED** [review]
LGTM (after rebase)

**Review by easeev — APPROVED** [review]
LGTM

**Pull request merged**